scripts/nfs-export-updater

```diff
@@ -49,13 +113,29 @@ You can now modify and update individual packages by running 'bitbake <package>'
     else:
         logging.getLogger().setLevel(logging.INFO)
 
+    import bb.tinfoil
+    with bb.tinfoil.Tinfoil() as tinfoil:
+        tinfoil.prepare(quiet=1)
+
+        b_topdir = tinfoil.config_data.getVar('TOPDIR')
+        b_machine = tinfoil.config_data.getVar('MACHINE')
+
+    # figure out default nfsroot directory to use
+    # will be of form ${TOPDIR}/nfsroot-{rootfs}-${MACHINE}'
+    if not args.exportdir:
+        args.exportdir = '%s/nfsroot-%s-%s' % (b_topdir, args.rootfs, b_machine)
+        logging.info(f"Using {args.exportdir} as nfsroot export dir")
+
     if not os.path.isabs(args.exportdir):
         args.exportdir = os.path.abspath(args.exportdir)
 
     if not os.path.exists(args.exportdir):
-        logger.error(f"{args.exportdir} does not exist.")
-        logger.error("Did you run runqemu-extract-sdk to create it?")
-        sys.exit(1)
+        logger.info(f"{args.exportdir} does not exist. Creating it..")
+
+        create_nfs_exportdir(args.rootfs, args.exportdir)
+
+    setup_nfs_exporter(args.exportdir)
```

Comment:
I'm wondering if we should try stopping a previously started unfsd, first?
```suggestion
    try:
        stop_nfs_exporter(args.exportdir)
    except CalledProcessError:
        pass
    
    setup_nfs_exporter(args.exportdir)
```

Even with the except/finally change below, a dying SSH connection might prevent the call to `stop_nfs_exporter()`. I just ran into this (although it would not have happened if I had run nfs-export-updater in a tmux/screen session before..).

Reply:
Oh, that's not so easy, because we cannot know the `nfs_instance` needed to stop it. Grepping that from the process list or deducing that from the files in `~/.runqemu-sdk/` seems hacky, maybe it's just not worth it.